tools/geospatial.go

```diff
@@ -43,18 +43,19 @@ type xyzPoint struct {
 	z float64
 }
 
-var unitConsistencyMap map[string]string = map[string]string{
-	"English Units": "US survey foot",
-	"SI Units":      "metre"}
+
+var unitConsistencyMap [][]string = [][]string{{"english units", "us survey foot", "foot_us", "foot us", "us foot"}, {"si units","metre", "meter"}}
```

Comment:
Don't think this needs to be `[][]string`, it could just be `[]string`. That would eliminate the need for the for loop below. Also, update the variable name, as it is no longer a map.

Reply:
I am checking if two units are on the same array. There are SI units and English Units array, if units are not in the same list then the unit is not consistent. Hence the need for an array of arrays.

I will change the name.